appveyor.yml

```diff
@@ -0,0 +1,29 @@
+version: 1.0.{build}
+pull_requests:
+  do_not_increment_build_number: true
+branches:
+  except:
+  - coverity_scan
+  - gh_pages
+skip_tags: true
+image:
+  - Visual Studio 2017
+configuration:
+  - Debug
+  - Release
+platform:
+  - x64
+  - Win32
+init:
+  - cmd: |
+      cmake -version
+before_build:
+  - cmd: |
+      cd /d %APPVEYOR_BUILD_FOLDER%
+      if "%PLATFORM%"=="Win32" mkdir build & cd build & cmake ..
+      if "%PLATFORM%"=="x64" mkdir build & cd build & cmake ..

```

Comment:
There is no difference between the handling of the two platforms. And I also think that we are in the right directory when the command is executed. So I think that
```yaml
before_build:
  - cmd: |
      mkdir build
      cd build
      cmake ..
```
would be enough (and even easier to read perhaps).